roles/openshift_health_checker/library/aos_version.py

```diff
@@ -47,8 +49,6 @@ def bail(error):  # pylint: disable=missing-docstring
 
     # determine what level of precision we're expecting for the version
     expected_version = module.params['version']
-    if expected_version.startswith('v'):  # v3.3 => 3.3
-        expected_version = expected_version[1:]
```

Comment:
If another check decides to use this ansible module, whoever gets to write that should have a clue that there are certain expectations about the version format.

I think the best chance we have of someone finding the information would be to have it live in the module itself?

Other than that, have the module be more defensive and make assertions about the input format.
And tests :-)